packages/client/src/v2-events/features/events/actions/print-certificate/Review.tsx

```diff
@@ -179,19 +182,31 @@ export function Review() {
     ))
 
     if (confirmed) {
-      handleCertify?.()
-      actions.printCertificate.mutate({
-        eventId: fullEvent.id,
-        data: form,
-        transactionId: uuid(),
-        type: ActionType.PRINT_CERTIFICATE
-      })
-      clear()
-      navigate(ROUTES.V2.EVENTS.OVERVIEW.buildPath({ eventId }))
+      setPrintingInProgress(true)
+      actions.printCertificate.mutate(
+        {
+          eventId: fullEvent.id,
+          data: { ...formValues, templateId },
+          transactionId: uuid(),
+          type: ActionType.PRINT_CERTIFICATE
+        },
+        {
+          onSuccess: () => {
+            handleCertify?.()
```

Comment:
```suggestion
           await handleCertify()
```

Reply:
This results in pdf print window not opening.